camunda.Dockerfile

```diff
@@ -4,82 +4,18 @@
 # see https://docs.docker.com/build/buildkit/#getting-started
 # Both ubuntu and eclipse-temurin are pinned via digest and not by a strict version tag, as Renovate
 # has trouble with custom versioning schemes
-ARG BASE_IMAGE="ubuntu:noble"
-ARG BASE_DIGEST="sha256:a08e551cb33850e4740772b38217fc1796a66da2506d312abe51acda354ff061"
-ARG JDK_IMAGE="eclipse-temurin:21-jdk-noble"
-ARG JDK_DIGEST="sha256:c04e695e59a97337e87d55ebbe9f527aacec1504b78ffac2730475057a8ea465"
+ARG BASE_IMAGE="reg.mini.dev/openjre:21-dev"
+ARG BASE_DIGEST="sha256:80e113714c426176d4c8159d0729bfbe799784f04e4efac4b376a9f17e0fa689"
+ARG JDK_IMAGE=""
```

Comment:
I don't find any usages of this ARG anymore inside this file, so we should seek to remove it.